app/views/recipes/_template_field.html.erb

```diff
@@ -0,0 +1,21 @@
+<div data-controller="dynamic-form-field visibility"
+     data-dynamic-form-field-index-value="<%= form.index %>"
```

Comment:
note that you can access the `fields_for` `index` attribute via `form.index` since `form` in this context is the ingredient's form